spec/requests/users_request_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users'
+      expect(response).to have_http_status(:success)
+      expect(response.status).to eq(200)
+      expect(response).to render_template('index')
+      expect(response.body).to include('The list of users')
+    end
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better to write more contextual tests that are descriptive. We can divide these expectations into different blocks.

  ```
  before(:each) do
    get '/users'
  end

  it 'returns HTTP success' do
    expect(response).to have_http_status(:success)
  end

  it 'returns OK status' do
    expect(response.status).to eq(200)
  end
  
  it 'renders the index template' do
    expect(response).to render_template('index')
  end
  
  it 'renders page with placeholder text' do
    expect(response.body).to include('The list of users')
  end
  ```

Reply:
will do Thank you

Reply:
Will Do 👍